tgui/packages/tgui/interfaces/Changelog.js

```diff
@@ -22,7 +22,6 @@ const icons = {
   soundadd: { icon: 'tg-sound-plus', color: 'green' },
   sounddel: { icon: 'tg-sound-minus', color: 'red' },
   add: { icon: 'check-circle', color: 'green' },
-  expansion: { icon: 'check-circle', color: 'green' },
```

Comment:
Uh, no, this is going to break old changelogs and the changelog is not exclusively used for GBP. What the hell, dude.

Reply:
my b

Reply:
@Iamgoofball You can give this exact feedback without making it seem hostile.

> ~~Uh, no,~~ this is going to break old changelogs and the changelog is not exclusively used for GBP ~~. What the hell, dude.~~